devops/cts_exclude_filter

```diff
@@ -1,31 +1,3 @@
-context
-device
-event
-exceptions
-kernel
-multi_ptr
-platform
-queue
-reduction
-optional_kernel_features
 accessor
-accessor_legacy
-vector_alias
-vector_api
-vector_constructors
-vector_load_store
-vector_operators
-vector_swizzle_assignment
-vector_swizzles
```

Comment:
FYI: some of these tests (e.g. vector_*) are in this list because they take a lot of time to compile and execute.